wpg/pg.go

```diff
@@ -41,6 +41,10 @@ type Column struct {
 	Type string `db:"data_type"json:"type"`
 }
 
+func (c Column) String() string {
```

Comment:
maybe overkill to add this function, happy to replace it with just doing the escaping inline